src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorWorker.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+#include <string>
+#include "RSU_MIB_4_1.h"
+#include <map>
+#include <vector>
+#include <memory>
+#include <algorithm>
+#include <nmeaparse/nmea.h>
+#include "PluginLog.h"
+#include <jsoncpp/json/json.h>
+#include "SNMPClient.h"
+#include <boost/algorithm/string/predicate.hpp>
+
+using namespace std;
+using namespace tmx::utils;
+using namespace tmx::utils::rsu41::mib::oid;
+
+namespace RSUHealthMonitor
+{
+    enum class RSUMibVersion
+    {
+        UNKOWN_MIB_V = 0,
+        RSUMIB_V_4_1 = 1,
+        RSUMIB_V_1218 = 2
+    };
+
+    struct RSUFieldOIDStruct
+    {
+        string field;
+        string oid;
+        bool required; // Indicate whether this field is required to before broadcasting the RSU status.
+    };
+
+    /**
+     * RSUStatusTable is custome defined RSU status information.
+     * The fields are the subset of fields from RSU MIB definition https://github.com/certificationoperatingcouncil/COC_TestSpecs/blob/master/AppNotes/RSU/RSU-MIB.txt
```

Comment:
The fields are a subset of the fields from the RSU MIB definition used to quantify the health of the RSU.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/564/commits/a49366ea8030afa8c668ec5879cf8e2d5cf69065